OsmAnd/src/net/osmand/plus/mapcontextmenu/builders/cards/ImageCard.java

```diff
@@ -509,6 +509,7 @@ private void getPicturesForPlace(List<ImageCard> result, long l) {
 					"Requesting location images...", false, false);
 			try {
 				if (!Algorithms.isEmpty(response)) {
+					//TODO extract place id
 					JSONArray obj = new JSONObject(response).getJSONArray("objects");
 					JSONArray images = ((JSONObject) ((JSONObject) obj.get(0)).get("images")).getJSONArray("outdoor");
```

Comment:
Here we need to fix pottential BUG! there is no clear evidence that 
```((JSONObject) obj.get(0))```
has no crash!

Reply:
try catchblock will handle the exception